apply/current_cluster.go

```diff
@@ -41,7 +41,7 @@ func listNodes() *corev1.NodeList {
 	}
 	nodes, err := client.ListNodes(c)
 	if err != nil {
-		logger.Info("current cluster nodes not found, will create a new cluster")
+		logger.Info("current cluster nodes not found, will create a new cluster %v", err)
```

Comment:
Why to use an Info level?

Reply:
When we create a new cluster, the listNodes will return error because current cluster is not exist, but it's normal.